src/routes/console/project-[project]/settings/updateServices.svelte

```diff
@@ -0,0 +1,127 @@
+            addNotification({
+                type: 'error',
+                message: error.message
+            });
+            trackError(error, Submit.ProjectService);
+        }
+    }
+
+    async function toggleAllServices(status: boolean) {
+        if (status && !showEnableAll) {
+            showEnableAll = true;
+            return;
+        }
+        if (!status && !showDisableAll) {
+            showDisableAll = true;
+            return;
+        }
+
+        try {
+            const path = '/projects/' + $project.$id + '/service/all';
+            await sdk.forConsole.client.call(
+                'PATCH',
+                new URL(sdk.forConsole.client.config.endpoint + path),
+                {
+                    'X-Appwrite-Project': sdk.forConsole.client.config.project,
+                    'content-type': 'application/json'
+                },
+                {
+                    status: status
+                }
+            );
+            invalidate(Dependencies.PROJECT);
+            addNotification({
+                type: 'success',
+                message:
+                    'All services for ' +
+                    $project.name +
+                    ' has been ' +
+                    (status ? 'enabled.' : 'disabled.')
+            });
+            trackEvent(Submit.ProjectService);
+        } catch (error) {
+            addNotification({
+                type: 'error',
+                message: error.message
+            });
+            trackError(error, Submit.ProjectService);
+        } finally {
+            showDisableAll = false;
+            showEnableAll = false;
+        }
+    }
+</script>
+
+<CardGrid>
+    <Heading tag="h6" size="7">Services</Heading>
+    <p class="text">
+        Choose services you wish to enable or disable for the client API. When disabled, the
+        services are not accessible to client SDKs but remain accessible to server SDKs.
+    </p>
+    <svelte:fragment slot="aside">
+        <ul class="buttons-list u-main-end">
+            <li class="buttons-list-item">
+                <Button text={true} on:click={() => toggleAllServices(true)}>Enable all</Button>
+            </li>
+            <li class="buttons-list-item">
+                <Button text={true} on:click={() => toggleAllServices(false)}>Disable all</Button>
+            </li>
+        </ul>
+        <FormList>
+            <form class="form card-separator">
+                <ul class="form-list is-multiple">
+                    {#each $services.list as service}
+                        <InputSwitch
+                            label={service.label}
+                            id={service.method}
+                            bind:value={service.value}
+                            on:change={() => {
+                                serviceUpdate(service);
+                            }} />
+                    {/each}
+                </ul>
+            </form>
+        </FormList>
```

Comment:
There seems to be too much spacing between the action rows and the form list:

<img width="1190" alt="image" src="https://github.com/appwrite/console/assets/1477010/5e2f3635-b59a-46c5-b5fa-b4cb01f90d4e">